cla_backend/apps/cla_eventlog/management/commands/find_and_delete_old_cases.py

```diff
@@ -0,0 +1,28 @@
+from django.core.management.base import BaseCommand
+from dateutil.relativedelta import relativedelta
+
+from legalaid.models import Case
+from cla_butler.tasks import DeleteOldData
+
+
+class FindAndDeleteCasesUsingCreationTime(DeleteOldData):
+    def get_eligible_cases(self):
+        super(FindAndDeleteCasesUsingCreationTime, self)._setup()
+        two_years = self.now - relativedelta(years=2)
+
+        return Case.objects.filter(created__lte=two_years).exclude(log__created__gte=two_years)
+
+
+class Command(BaseCommand):
+    help = (
+        "Find or delete cases that are 2 years old or over that were not deleted prior to the task command being fixed"
+    )
+
+    def handle(self, *args, **kwargs):
```

Comment:
What's the default behaviour if I don't pass either "find" or "delete"? Is this desirable? What *should* the default behaviour be?

Reply:
Thanks, I've made the default `find` since that is what we would want to do first before any deletions.

Reply:
Perfect